backend/substrapp/compute_tasks/outputs.py

```diff
@@ -107,6 +106,16 @@ def _save_model_to_local_storage(self, model: context.OutputResource) -> dict:
             },
         }
 
+    @retry(raise_exception=True)
+    def _register_models_in_api(self, registered_models: dict):
+        return organization_client.post(
+            self._ctx.channel_name,
+            settings.LEDGER_MSP_ID,
+            settings.DEFAULT_DOMAIN + reverse("api:model-list"),
+            data={},
+            files={"metadata": json.dumps(registered_models)},
```

Comment:
You could use data now that you just send a JSON, no ?

Reply:
I have to check and test, because it's a nested dict  !

Reply:
https://stackoverflow.com/questions/48193316/passing-a-nested-dictionary-to-requests-module
Does not seem possible, we have to pass under `json` arg or json dump the data as I did